code/game/gamemodes/miniantags/revenant/revenant_abilities.dm

```diff
@@ -260,7 +260,7 @@
 				for(var/obj/machinery/dna_scannernew/dna in T.contents)
 					dna.open_machine()
 				for(var/obj/structure/window/window in T.contents)
-					window.hit(rand(30,80))
+					window.take_damage(rand(30,80))
```

Comment:
standardization of all procs handling structural damage. All machines and structures now use "take_damage()".